### PR TITLE
fix(typos): Argosy Hijacking

### PR DESCRIPTION
**Pull request by TheGiraffe3**
**Typo fix**

This PR addresses the bug/feature described in issue
None

## Summary
Fixes a few bugs in the `Argosy Hijacking` mission. Also makes the `npc` tags be in alphabetical order. 

## Screenshots
None. 

## Usage examples
N/A

## Testing Done
None. 

## Save File
This save file can be used to test these changes:
None, sorry.  

## Artwork Checklist
N/A

## Wiki Update
N/A

## Performance Impact
N/A

### CODE REVIEW

**Comment by TheGiraffe3** [comment]
@tibetiroka will you review this?

**Comment by TheGiraffe3** [comment]
This PR is ready for review/merge.

File: data/human/human missions.txt [hunk 1]
```diff
@@ -5984,6 +5984,7 @@ mission "Argosy Hijacking"
 		on kill
 			set "destroyed hijacked argosy"
 			dialog `You have destroyed the Moonshake Express. The ship's captain won't be impressed.`
+			fail
```
Comment (Saugia) [inline data/human/human missions.txt:5987]:
```suggestion
```
From the wiki:

`save: The mission fails if the given NPC is destroyed.`

I strongly recommend playtesting any and all mission changes to see if the mission is actually failing or not/if the current syntax already is correct.

Comment (TheGiraffe3) [reply data/human/human missions.txt:5987]: Forgot about the save. You're correct.

**Pull request merged**